tests/conftest.py

```diff
@@ -38,6 +38,10 @@ def pytest_configure(config):
     os.environ["OPENSEARCH_ENRICH"] = "no"
     os.environ["ASYNC_TASKS_KAFKA_BOOTSTRAP_SERVERS"] = "kafka01.test:6668"
 
+    # For testing job_tracker_v2: TestYarnStatusGetter and TestYarnJobTracker
+    os.environ["YARN_REST_API_BASE_URL"] = "https://openeo.test"
+    os.environ["YARN_AUTH_USE_KERBEROS"] = "no"
```

Comment:
Setting `YARN_REST_API_BASE_URL` is unnecessary if you pass the URL to the constructor (see above).
Setting `YARN_AUTH_USE_KERBEROS` is unnecessary if you're not doing any actual HTTP requests.

Reply:
I was indeed able to eliminate them by following your other suggestions.